configure.py

```diff
@@ -36,36 +36,36 @@ def parse_arguments():
       help="Build with ENABLE_LLD=ON (optional)",
       dest="enable_lld",
       default=True,
-      action=argparse.BooleanOptionalAction,
+      action="store_true",
```

Comment:
could you briefly explain why you need this ?

Reply:
This is to support python 3.7. If there is any strong reason to have `argparse.BooleanOptionalAction`, I can revert. But I would say if we can support 3.9 and 3.7 that would be better.

Reply:
ok with me but please extract this as a standalone PR that should land separately.

Reply:
Done! https://github.com/google/iree-llvm-sandbox/pull/258